doc/lightning-pay.7.txt

```diff
@@ -37,11 +37,10 @@ The response will occur when the payment fails or succeeds.  Once a
 payment has succeeded, calls to *pay* with the same 'bolt11' will
 succeed immediately.
 
-The command will keep finding routes and retrying the payment until
-it succeeds, or the given 'retry_for' seconds passes.
-Note that the command may stop retrying while a pending payment is
-ongoing, which you need to monitor with *listpays*.
-'retry_for' defaults to 60 seconds and can only be an integer.
+Until 'retry_for' seconds passes (default: 60), the command will keep
+finding routes and retrying the payment.  However, a payment may be
```

Comment:
Suggest breaking at each sentence (so that rewording an earlier sentence in a paragraph will not affect succeeding sentences).